dubbo-common/src/main/java/org/apache/dubbo/common/config/ConfigurationUtils.java

```diff
@@ -38,7 +38,7 @@
     public static int getServerShutdownTimeout() {
         int timeout = Constants.DEFAULT_SERVER_SHUTDOWN_TIMEOUT;
         Configuration configuration = Environment.getInstance().getConfiguration();
-        String value = configuration.getString(Constants.SHUTDOWN_WAIT_KEY);
+        String value = StringUtils.trim(configuration.getString(Constants.SHUTDOWN_WAIT_KEY));
```

Comment:
Do we need to add any UT for this scenario ?

Reply:
+1 to have UT for org.apache.dubbo.common.config.ConfigurationUtils#getServerShutdownTimeout

Reply:
Sure, I will do that.